Jbarget/oxymore
# adds assets and content for Map project

### What changes have you made?

- adds a first pass at the zoomable map
- i imagine theres more work to be done but happy with this as a start

### Which issue(s) does this PR fix?

Fixes #16

### Screenshots (if there are design changes)
before zooming:
![image](https://user-images.githubusercontent.com/11725595/103567950-f5d70c80-4ec4-11eb-9259-76a77c4a7ce8.png)

after zooming: 
![image](https://user-images.githubusercontent.com/11725595/103567971-fff90b00-4ec4-11eb-899a-880d3097bef2.png)


### How to test

- adjust launch date to show actual map
- go to map project
- should be able to zoom/scroll on map
